.github/workflows/build-wheels.yml

```
@@ -0,0 +1,75 @@
+name: Build wheels
+
+on:
+  push:
+    branches: [ master ]
```

```suggestion
    branches: [ develop ]
```

Same below.

We use master for releases. PRs get merged onto develop, so we should be building from there.